docs/tutorial/Blocking with Anonlink Entity Service.ipynb

```diff
@@ -0,0 +1,1783 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "show_clk_popcounts(\"clks_b.json\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Blocking\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We create a blocking schema to group the encodings into similar blocks on the client side.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 19,
+   "metadata": {
+    "pycharm": {
+     "is_executing": true
+    }
+   },
+   "outputs": [],
+   "source": [
+    "# NBVAL_IGNORE_OUTPUT\n",
+    "with open(\"blocking_schema.json\", 'wt') as f:\n",
+    "    f.write(\"\"\"\n",
+    "{\n",
+    "    \"type\": \"lambda-fold\",\n",
+    "    \"version\": 1,\n",
+    "    \"config\": {\n",
+    "        \"blocking-features\": [6],\n",
+    "        \"Lambda\": 3,\n",
+    "        \"bf-len\": 64,\n",
+    "        \"num-hash-funcs\": 3,\n",
+    "        \"K\": 5,\n",
+    "        \"input-clks\": true,\n",
+    "        \"random_state\": 0\n",
+    "    }\n",
+    "}    \n",
+    "    \"\"\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 20,
+   "metadata": {
+    "pycharm": {
+     "is_executing": true
+    }
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Number of Blocks:   96\r\n",
+      "Minimum Block Size: 2\r\n",
+      "Maximum Block Size: 1152\r\n",
+      "Average Block Size: 156\r\n",
+      "Median Block Size:  85\r\n",
+      "Standard Deviation of Block Size:  188.9700616778391\r\n"
+     ]
+    }
+   ],
+   "source": [
+    "!anonlink block 'clks_a.json' 'blocking_schema.json' 'blocks_a.json'"
```

Comment:
I think it is better to add a paragraph to brief talk about the statistics block command generate and what do they indicate. For example, the running time will be largely dominated by the maximum block size, the smaller the average block size, the better, etc. This can help users understand how block help them scale up the record linkage.